<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>135</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn33">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn14">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn19">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn10">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn32">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn22">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn29">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(0, 98, 204);
font: 12pt &quot;Poppins&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>⌫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn18">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn27">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>ñ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn23">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn35">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(0, 98, 204);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>⇨</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn30">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(0, 98, 204);
font: 11pt &quot;Poppins&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>?123</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn17">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn15">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn28">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn25">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn16">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn21">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(0, 98, 204);
font: 11pt &quot;Poppins&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mayús</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn31">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn26">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn34">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn13">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn24">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>10</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 11pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
